### ISSUE TITLE
Questions with SM AutoDemo

### ISSUE THREAD

**Issue by crashzk** [CrazyHackGUT/sm-autodemo#18]
I would like to start using this plugin to manage the demos on my servers, but there were some questions.

I can configure using the **`AutoDemo_FTPUpload`** plugin to upload the demos after it's finished to my web hosting, correct? So players could go and download the demo after the upload is complete.

About SourceTV setup:

- tv_enable 1
- tv_autorecord 0
- tv_maxclients 0
- tv_name "AutoDemo"
- tv_port 27020

Can I freely change the **`tv_maxclients`**, **`tv_name`** and **`tv_port`** options?

**Comment by CrazyHackGUT** [comment]
> I can configure using the AutoDemo_FTPUpload plugin to upload the demos after it's finished to my web hosting, correct?

Yes, but you need manually implement listing. This can be reached via built-in feature "auto-index" in web-servers, for example.
https://nginx.org/en/docs/http/ngx_http_autoindex_module.html
https://httpd.apache.org/docs/2.4/mod/mod_autoindex.html

> Can I freely change the tv_maxclients, tv_name and tv_port options?

Yes, of course. Plugin limits only `tv_autorecord` value: they should be zero always.
